scss/components/_inline-datepicker.scss

```diff
@@ -0,0 +1,49 @@
+$form-control-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+$inline-error-bg: #fbe3e4;
+
+.civicase-inline-datepicker__wrapper {
+  [civicase-inline-datepicker] {
+    border: 1px solid transparent;
+    border-radius: $input-border-radius;
+    height: $input-height-base;
+    padding: $padding-base-vertical $padding-base-horizontal;
+    width: 140px !important;
+  }
+
+  .form-control {
+    box-shadow: none;
+    display: inline-block;
+    margin-left: -($padding-base-horizontal + 1px);
+  }
+
+  .form-control.ng-invalid {
+    background-color: $inline-error-bg;
+  }
+
+  .addon {
+    margin-top: -3px !important;
+    opacity: 0;
+    transition: opacity 0.15s;
```

Comment:
Do we need this new transition? if it is not present for other shoreditch elements, it should not be for this one too.

Reply:
because:

1 - The `.addon` element is not a bootstrap element, it comes from jQuery UI.
2 - There's a CiviCRM definition in Shoreditch for this element: https://github.com/civicrm/org.civicrm.shoreditch/blob/2ff49b9f43f89b9e6446575daf223be9f1dcc04f/scss/civicrm/common/_forms.scss#L320-L333 but it doesn't handle the same transition as the opacity in the box shadow of a form-controller because in CiviCRM they are not paired together. CiviCRM uses a normal input.
3 - The `0.15s` is a constant because there's no variable definition for it: https://github.com/civicrm/org.civicrm.shoreditch/blob/master/base/scss/vendor/bootstrap/_forms.scss#L146